### DISCUSSION TITLE
Is there a way to detect the Bluetooth status changes?

### DISCUSSION THREAD

**Discussion by juniorsaraviao** [capacitor-community/bluetooth-le#624] [Q&A] [answered]
We have `isEnabled()` to detect whether Bluetooth is on/off. But if a device is connected and then the Bluetooth is turned off, there's no way to detect that change.

For example the BleClient.connect has a callback to verify if the device is disconnected.
`connect(deviceId: string, onDisconnect?: (deviceId: string) => void, options?: TimeoutOptions)`

I'm currently using bluetooth-le: 3.1.1

**Comment by pwespi** [comment, marked as answer]
There is [`startEnabledNotifications`](https://github.com/capacitor-community/bluetooth-le?tab=readme-ov-file#startenablednotifications).

**Reply by juniorsaraviao** [reply]
Hi @pwespi, I was using the `startenablednotifications` but I realized it returned two values: The first is always **false** and the second is the actual **Bluetooth state** (it happens only on Android 10 and above devices). Is that an expected behavior?

| Issue |
| ------------- |
| <video src="https://github.com/juniorsaraviao/BluetoothSample/assets/43689290/2c2c6a8d-da8d-4638-bd95-e0772680d4a3" width=180 height=392> |

**Reply by juniorsaraviao** [reply]
would it be good if I created a bug for the issue above?
